liqd_product/apps/partners/templates/partner_base.html

```diff
@@ -0,0 +1,38 @@
+{% extends 'base.html' %}
+
+{% load userindicator static i18n %}
+
+{% block after_header %}
+<img src="{{partner.logo}}" alt="Logo" />
```

Comment:
I am not sure about the `alt` attribute. A semantically correct alt text would be `partner.name`, but that is displayed next anyway. So just leave it empty? Then again, this will probably become a "home" link, so it should contain text. @vellip what do you think?